### DISCUSSION TITLE
Is it possible to search for specific phrases among all versions of a file?

### DISCUSSION THREAD

**Discussion by YousufSSyed** [community/community#51916] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

A while ago I had the phrase `backlinks` in a CSS file, but don't have it anymore. I want to bring it back and would like to search the phrase `backlinks` and see all instances of it across versions of the file.

**Comment by look** [comment, marked as answer]
GitHub code search doesn't currently support searching history, so you'd need to clone your repository to do this. A command like this will search the commits on your current branch and show you matching diffs:

```
git log -p -S backlinks
```

Hope this helps. The `-S` operator is called the pickaxe and you can [read more about it here](https://git-scm.com/book/en/v2/Git-Tools-Searching#_git_log_searching).
